script.js

```diff
@@ -138,9 +138,10 @@ openModal.forEach(function (e) {
     button.innerHTML = "Search available times";
     button.type = "submit";
     modal.appendChild(button);
+    const selectedDate = inputDate.value;
     button.addEventListener("click", async function () {
       const res = await fetch(
-        `https://lernia-sjj-assignments.vercel.app/api/booking/available-times?date=${inputDate.value}&challenge=3"`
+        `https://lernia-sjj-assignments.vercel.app/api/booking/available-times?date=${selectedDate}&challenge=3"`
```

Comment:
The base of this long URL reoccurs several times. Maybe it could be stored in a variable somewhere and reused?